lib/IRGen/GenType.cpp

```diff
@@ -1125,13 +1126,17 @@ TypeConverter::TypeConverter(IRGenModule &IGM)
   if (IGM.IRGen.Opts.EnableResilienceBypass)
     LoweringMode = Mode::CompletelyFragile;
 
+  auto libraryImportPath = IGM.Context.SearchPathOpts.RuntimeLibraryImportPath;
+  llvm::SmallString<128> defaultPath(libraryImportPath);
+  llvm::sys::path::append(defaultPath, "layouts.yaml");
```

Comment:
You can't assume this'll always be there; it won't on Linux, for example.

Reply:
Should we enumerate somewhere which platforms have this and which ones do not? I don't want to just silently continue if the file is not present, in case it gets lost in the installation process somehow.

Reply:
Sure, that sounds fine to me. (Naming comment still applies.)